deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+docker build --target prod -t rysharp/todo-app:${TRAVIS_COMMIT} .
+docker push rysharp/todo-app:${TRAVIS_COMMIT}
```

Comment:
Minor, but we're not ever pushing to the `:latest` tag, which is the default one, so somebody wanting to run the most up to date version of your code would need to track down the appropriate commit hash